homeassistant/components/deconz/__init__.py

```diff
@@ -186,3 +187,92 @@ def deconz_shutdown(event):
         entity_picture="/static/images/logo_deconz.jpeg",
         submit_caption="I have unlocked the gateway",
     )
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class DeconzFlowHandler(config_entries.ConfigFlowHandler):
+    """Handle a deCONZ config flow."""
+
+    VERSION = 1
+
+    def __init__(self):
+        """Initialize the deCONZ flow."""
+        self.bridges = []
+        self.deconz_config = {}
+
+    async def async_step_init(self, user_input=None):
+        """Handle a flow start."""
+        from pydeconz.utils import async_discovery
+
+        if DOMAIN in self.hass.data:
```

Comment:
So you cannot create a config entry if you have one configured, but you will be able to configure a host after you have created a config entry?

Reply:
If there is an instance of deconz running, the config flow won't run. What do you mean by "but you will be able to configure a host after you have created a config entry"?